test/no-accessor-recursion.js

```diff
@@ -109,6 +109,14 @@ test.snapshot({
 				}
 			};
 		`,
+		// Static getter
+		outdent`
+			class Foo{
+				static get bar() {
+					return this.bar;
+				} 
+			}
+		`,
```

Comment:
Shouldn't this invalid? I only asked to test, didn't mean to ignore.

Reply:
> Shouldn't this invalid? I only asked to test, didn't mean to ignore.

Well, I thought this didn't have infinite recursion until I tried it